README.md

```
@@ -1 +1,179 @@
-# SeCause-BE
\ No newline at end of file
+# SeCause Backend
+
+<div align="center">
+
+### AI 기반 코드 보안 취약점 분석 서비스의 백엔드 API
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)